certlogic/specification/README.md

```diff
@@ -215,11 +215,10 @@ A date-time offset operation has the following form:
         ]
     }
 
-A time unit (string) can be one of: "year", "month", "day", "hour".
-This operation is the *only* way to construct date-time values.
-Offsetting a date-time isn't affected by daylight saving time (DST transitions), nor by leap seconds.
-
-To convert a date(-time) string without offsetting, specify an amount of `0`, and any time unit.
+A time unit is one of the following string values: "year", "month", "day", "hour".
+This operation is the *only* way to construct date-time values in CertLogic.
+Offsetting a date-time isn't affected by daylight saving time (DST transitions), nor by leap years or leap seconds.
+To convert a date(-time) string to a date-time value, specify an amount of `0`, and any time unit.
```

Comment:
Maybe also explicitly mention that `plusTime` especially does not permit other date-time *values*. As such operations like `plusTime(plusTime("...", 0, "hour"), 10, "day")` are not possible.

Reply:
Good suggestion! Too bad I already pulled the trigger on this PR :) I'll make a new branch for this, although I might not immediately promote it to a PR, in case we can combine this with some more clarifications we make in the short term.